### ISSUE TITLE
Feature Request: Frame-By-Frame Segmentation Tagging

### ISSUE THREAD

**Issue by nimroha** [Labelbox/Labelbox#90]
I have a short videos (15s) where I need a segmentation of a certain object in all frames.

Currently I split the video into frames and simply uploaded all the images to one project.
But, in order to retain my labelers' sanity, it would be great if each image would load with the segmentation from the previous image.
Then, they could take advantage of your vertex dragging feature, since changes between frames should be small.

PS. It would also be great if down the road I could simply upload a video file and you'll do the splitting ;)

**Comment by manuaero** [comment]
Yeah, really cool idea @nimroha . I'd love to speak with you and discuss more of this. Can you please ping me at manu@labelbox.com?
